packages/derived-wallet-ethereum/src/EIP1193DerivedPublicKey.ts

```diff
@@ -84,6 +87,10 @@ export class EIP1193DerivedPublicKey extends AccountPublicKey {
     return recoveredAddress === this.ethereumAddress;
   }
 
+  async verifySignatureAsync(args: { aptosConfig: AptosConfig, message: HexInput, signature: Signature }): Promise<boolean> {
```

Comment:
I am surprised it lets you pass  `aptosConfig: AptosConfig` when `this.verifySignature` does not expect this property

Reply:
Yeah, if you explicitly define it it'll complain but if you pass the whole object its chill

Reply:
can we be more specific here and only pass the expected props?
```
const { message, signature } = args
this.verifySignature({message, signature})
```